nix/tools/withTools.nix

```diff
@@ -25,7 +25,6 @@ let
             "ARG_USE_ENV([PGUSER], [postgrest_test_authenticator], [Authenticator PG role])"
             "ARG_USE_ENV([PGDATABASE], [postgres], [PG database name])"
             "ARG_USE_ENV([PGRST_DB_SCHEMAS], [test], [Schema to expose])"
-            "ARG_USE_ENV([PGRST_DB_ANON_ROLE], [postgrest_test_anonymous], [Anonymous PG role])"
```

Comment:
Hm. I think the idea was to make the `posgrest-with-postgresql-xx` helpers more independent of the actual schema they are are loading. The files to load can be changed with `-f`, I think. And we do so for IO tests and loadtest.

I think I just forgot to add a db role setting in the spec schema for the anonymous role. The tests didn't fail, because the spec tests have a separate config for that anyway. But to be able to load the spec schema with `postgrest-run` it would be good to have that db setting.

And I'd go one step further: We should move the schema setting into the db config, too. No need to hardcode `test` here for all.